examples/federated_statistics/df_stats/visualization/adults_stats.json

```diff
@@ -0,0 +1,3734 @@
+{
```

Comment:
do we really want to check in this large JSON in repo?

another thing, this "visualization" folder is more like a "tutorial"

Should we separate "example" jobs from "end-to-end" tutorials?

Reply:
All of our examples result can be visualized in some way.
Like loss or accuracy can be written out into a tensorboard and visualize.

This is just a general discussion of whether we want to put these into the "examples" folder.
I am fine with putting things into examples folder.